src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -65,22 +65,22 @@ describe('Detailed view and edit a selected work item - ', () => {
     fakeUserList = [
       {
         attributes: {
-          fullName: 'Harry Potter',
-          imageURL: 'http://nerdist.com/wp-content/uploads/2016/02/20160210_nerdistnews_harrypottercursedchild_1x1.jpg'
+          fullName: 'WIDCT Example User 0',
```

Comment:
Don't you think keeping the name in first name - last name format will be a better, in case we decide to implement some test on that in future.